docs/ENVIRONMENT_VARIABLES.md

```diff
@@ -29,6 +29,18 @@ All of the environment variables used by this application conform to the [`djang
 | COLO_ACCOUNT_ADDRESS | The coin distribution address. | `str` | '' |
 | COLO_ACCOUNT_PRIVATE_KEY | The coin distribution private key. | `str` | '' |
 
+## EthOS
+
```

Comment:
might be worth explaining what this is here? its probably not relevant to 99% of ppl who fork the repo